### PR TITLE
JS API for execution.test.abort

### PR DESCRIPTION
**Pull request by codebien**
Docs for `test.abort()` new feature

### CODE REVIEW

File: src/data/markdown/docs/02 javascript api/06 k6-execution.md [hunk 1]
```diff
@@ -64,6 +65,14 @@ Meta information and execution details about the current running [scenario](/usi
 | iterationInTest     | integer | The unique and zero-based sequential number of the current iteration in the scenario. It is unique in all k6 execution modes - in local, cloud and distributed/segmented test runs. However, while every instance will get non-overlapping index values in cloud/distributed tests, they might iterate over them at different speeds, so the values won't be sequential across them. |
 
 
+### test
+
+Control the test execution.
+
+| Field | Type | Description |
+|-------|------|-------------|
+| abort | | It immediately aborts a k6 load test run with the exit code `108`. Aborting a test is fully supported in `k6 cloud`, preserving the same user experience for both local and cloud users. |
```
Comment (imiric) [inline src/data/markdown/docs/02 javascript api/06 k6-execution.md:74]: I'm wondering if "field" is the right column name... "Property" would be more accurate. Same goes for the other sections. It's unrelated to this PR, but we might as well change it here if you agree.

And perhaps the value of Type here could be "function"?

**Review by imiric — CHANGES_REQUESTED** [review]

**Review by imiric — APPROVED** [review]
Great! Thanks a lot for picking this up! :smile:

**Review by oleiade — APPROVED** [review]
Nickel Chrome 👍🏻 Cheers for that 🦖

**Review by na-- — APPROVED** [review]

**Comment by ppcano** [comment]
@codebien @na-- 

Because the default behavior of k6 is `same user experience for both local and cloud users`,  I suggest removing:

`Aborting a test is fully supported in k6 cloud, preserving the same user experience for both local and cloud users.`

And, only provide the details on the exceptions.

**Pull request merged**